CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## 0.26.0
 
 * Update the Open Policy Agent Authorizer to version [1.1.0](https://github.com/Bisnode/opa-kafka-plugin/releases/tag/v1.1.0)
+* Add support for building connector images from the maven coordinates
```

Comment:
```suggestion
* Add support for building connector images from the maven coordinates (#4179)
```

Reply:
Should `maven` be `Maven` with capital `M`?